### DISCUSSION TITLE
code blocks executions

### DISCUSSION THREAD

**Discussion by moolight-seashell** [AppFlowy-IO/AppFlowy#3554] [General help wanted] [answered]
is there a way to execute hi level interpredted laguage like python directly in code bloks like in jupyter notebook ? or it is planned to do it later ?

do the same things with compiled code is way harder but if we can have only python support in a first place it will be amaizing

**Comment by MayurSMahajan** [comment, marked as answer]
No, there is no such functionality to execute code within AppFlowy's CodeBlock. And I don't think it is planned, if it is something you need, make a new feature request. 
Side Note: Executing code within Codeblock in AppFlowy is not something that the codeblock is intended for.
